gcloud-workstation-phpstorm/scripts/011_set-default-shell.sh

```diff
@@ -42,9 +47,12 @@ zstyle ':completion:*' verbose true
 
 zstyle ':completion:*:*:kill:*:processes' list-colors '=(#b) #([0-9]#)*=0=01;31'
 zstyle ':completion:*:kill:*' command 'ps -u $USER -o pid,%cpu,tty,cputime,cmd'
+alias docker-compose='docker compose'
 EOF
+fi
 
-cat <<'EOF' >/root/.zshrc
+if [ ! -f /home/user/.zshrc ]; then
+    cat <<'EOF' >/home/user/.zshrc
 # Set up the prompt
```

Comment:
why not just `cp /root/.zshrc /home/user/.zshrc`? 
Then the block with the content of the file would only be here once, which makes it more maintainable.

EDIT: The fix should work, that looks good.

Reply:
the .zshrc from `root` is slightly different than the one from `user` since it has the PATH environment variable update for `mdless` and `root` does not. Didn't see it necessary to install `mdless` for root quite honestly. I would otherwise agree with that suggestion. I'd rather not have a wrong path added to `root`'s .zshrc.

Reply:
Ended up copying the root config file as suggested but not adding editing the PATH for both `root` and `user`. Reason is that `mdless` won't work as `root` without further changes (complains about libraries) that I didn't see the point of making since `user` should be the one using `mdless` anyway.